week0/pyomo_watertap_intro_tutorial.ipynb

```diff
@@ -0,0 +1,536 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "949ae5d5-d0b1-4269-87e6-29b8f3df0071",
+   "metadata": {},
+   "source": [
+    "# Introduction to Pyomo for WaterTAP Users\n",
```

Comment:
Considering that this one will pair with the one @MarcusHolly put up, we should consider renaming the file to something indicating a more detailed/advance intro to pyomo, perhaps along with a title like "Deeper dive into Pyomo for WaterTAP Users" and add a single sentence saying that this is meant to follow the other, simpler python/pyomo tutorial.